docs/camperbot/commands.md

```diff
@@ -2,18 +2,19 @@
-| `!theyknow`           | `!theyknow @someone`                | 🚧Shows a list of techs/languages the specified user knows                                                                        |
+| Commands              | Usage                                        | Description                                                                                                                       |
+| --------------------- | -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `!help`, `!h`         | `!help`, `!h !mine`                          | Either shows a help for a single command, or DMs you a help link if no parameters are specified.                                  |
+| `!marshmellow`, `!mm` | `!mm @someone`                               | Gives a user a marshellow for being helpful. Basically style points ✨                                                            |
+| `!mine`               | `!mine`                                      | Tells you how many marshmellows you have                                                                                          |
+| `!knowme`             | `!knowme`                                    | Mostly for testing purposes, adds you to the database's if you do not exist in it yet                                             |
+| `!repopulate`         | `!repopulate`                                | Mostly for testing purposes. Goes through the list of users in the server and adds unknown users to the database. **Firestarter** |
+| `!ping`               | `!ping`                                      | Mostly for testing purposes. Responds with `pong!`                                                                                |
+| `!camping`            | `!camping`                                   | 🚧Gives you the **Camper** role if you've been part of the discord for a month                                                    |
+| `!nexteevent`         | `!nextevent`                                 | 🚧Tells you when the next event is with a link                                                                                    |
+| `!iknow`              | `!iknow Ruby`, `iknow 'HTML & CSS'`          | 🚧Add a self-summary of languages/tech you know                                                                                   |
+| `!theyknow`           | `!theyknow @someone`                         | 🚧Shows a list of techs/languages the specified user knows                                                                        |
+| `!emoji`              | `!emoji rip https://i.imgur.com/w3duR07.png` | Adds a new emoji to the server                                                                                                    |
```

Comment:
When you add the role to the `help.json` file, you'll have to add `**Camper**` at the end of this